### ISSUE TITLE
Add support for target field `url_path`

### ISSUE THREAD

**Issue by Alexnortung** [strapi-community/strapi-plugin-url-alias#9]
### Summary

It should be possible to use the `url_path` field as a target field in other plugins such as sitemap and preview button.
Currently the entities that use url-alias, can only use `url_path_id` as their target field.

### Why is it needed?

If you use url-alias to set urls for your routes, it should also be possible to use it with other plugins such as sitemap and preview button.

### Suggested solution(s)

I'm not really sure how the internals work, but a solution would be to have `url_path` added as a field when the entity is fetched/queried.

**Comment by boazpoolman** [comment]
Yes! This is very important. I totally agree.

We have middleware to alter the API responses. So the content API includes the url_path.
Though we should also have this feature when we use the internal query layers like the entityService.

I've thought about this and I think we can fix this by including extra lifecycle methods to alter beforeFind for example.

**Comment by boazpoolman** [comment]
I've put together a PR that should tackle this issue completely.

Could you test it out for me? 
You can install it in your project like so:

```
yarn add strapi-community/strapi-plugin-url-alias#pull/10/head
npm install strapi-community/strapi-plugin-url-alias#pull/10/head
```

**Comment by boazpoolman** [comment]
This was released with version `1.0.0-alpha.5` of the plugin.
If you're still experiencing the issue beyond this version feel free to re-open the GH issue or create a new one.
